_docs/howto/configure/gsuite-group-collection.md

```diff
@@ -4,35 +4,44 @@ order: 203
 ---
 #  {{ page.title }}
 
-This page describes how to enable the collection of GSuite Google Groups for
+This page describes how to enable the collection of G Suite Google Groups for
 processing by Forseti Scanner and Enforcer.
 
-## Enable Domain-Wide Delegation in GSuite
+## Enable Domain-Wide Delegation in G Suite
 
-To enable collection of GSuite Google Groups, follow the steps below to create a
+To enable collection of G Suite Google Groups, follow the steps below to create a
 service account just for this functionality:
 
-1.  Create a service account to inventory GSuite Google Groups:
-    1.  Go to your
-        [Google Cloud Platform console](https://console.cloud.google.com/iam-admin/serviceaccounts) and
-        create a new service account.
-    1.  Click the "dot" menu on the service account row, then click "Edit". 
-        Check "Enable domain-wide delegation", then click "Save".
-    1.  Create and download the JSON key to your local environment.
-    1.  On the service account row, you should now see a "View Client ID" link. Click it.
-        Copy the "Client ID" value, which will be a large number.
-1.  Enable the service account in your GSuite admin control panel. You must have
+1.  Create a service account to inventory G Suite Google Groups:
+    1.  Go to
+        [Cloud Platform Console Service Accounts](https://console.cloud.google.com/iam-admin/serviceaccounts)
+        and click **Create service account**.
+    1.  On the **Create service account** dialog that appears, set up your service account:
+        1.  Enter a service account name.
+        1.  Select the **Enable G Suite Domain-wide Delegation** checkbox.
```

Comment:
Noting belatedly that we closed this one while experimenting with site preview and working in a branch locally after creating it in the web UI. The changes from this PR are published and I'm investigating the consent screen item noted above. I'll follow up on that separately.